src/ProfilesList.json

```
@@ -156,5 +156,6 @@
   "abdosahsah.json",
   "athenacats.json",
   "RaghavSaptam.json",
-  "JaredCaprio.json"
+  "JaredCaprio.json",
+  "Adebayoibrahim.json",
```

```suggestion
  "Adebayoibrahim.json"
```
If you are adding you filename at last than no need of the comma `,`. You can remove the comma.